website/content/docs/upgrading/index.mdx

```diff
@@ -8,6 +8,8 @@ description: |-
 
 # Upgrading Waypoint
 
+~> **Note:** `waypoint install` and `waypoint server install` do not support an upgrade. A `waypoint upgrade` command is under consideration for Waypoint 0.2. One way to use the latest server is to [remove the existing server](/docs/troubleshooting#remove-the-waypoint-server) and then reinstall with `waypoint install`.
```

Comment:
I'd make a note here about backing up.

Reply:
Added a reference to backup with sample `docker` and `kubectl` commands.